Segunda_entrega/Sintactico.y

```diff
@@ -221,18 +238,7 @@ else_: ELSE {
 condicion:
          comparacion 
          |OP_NOT comparacion{printf("NOT CONDICION\n");}
-         |comparacion op_and_ {	printf("CONDICION CON AND\n");
-											int i, x;
```

Comment:
Esto está mal porque la posición actual es el comienzo de la segunda condición.
Si la primera condición es falso y es un AND, tengo que saltar todo el cuerpo. Donde voy a tener que saltar solo lo voy a saber cuando procese ENDIF